web/impact/requirements/base.txt

```diff
@@ -34,7 +34,7 @@ Django==1.10.3
 djangorestframework
 drf-schema-adapter
 drf-tracking
-git+https://github.com/masschallenge/django-accelerator.git@2017.09.06a#egg=django-accelerator
+git+https://github.com/masschallenge/django-accelerator.git@AC-4812#egg=django-accelerator
```

Comment:
we willl need to conciously change this once the rollout happens. wondering if it makes sense to change this to the day of the release once we agree that it will happen on the given day.

Reply:
I think that once the PR in django-accelerator is approved, it can be merged in, and a release tag can be created there - and then we can safely update impact-api and accelerate.